internal/connectors/plugins/public/increase/payments.go

```diff
@@ -0,0 +1,85 @@
+package increase
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/formancehq/payments/internal/models"
+)
+
+// Using pollingState from accounts.go
+
+func (p *Plugin) FetchNextPayments(ctx context.Context, req models.FetchNextPaymentsRequest) (models.FetchNextPaymentsResponse, error) {
+	state, err := p.getPollingState(req.State)
+	if err != nil {
+		return models.FetchNextPaymentsResponse{}, fmt.Errorf("failed to get polling state: %w", err)
+	}
+
+	if state.LastFetch.Add(p.config.PollingPeriod).After(time.Now().UTC()) {
+		return models.FetchNextPaymentsResponse{}, nil
+	}
+
+	// Get succeeded transactions
+	transactions, nextCursor, hasMore, err := p.client.GetTransactions(ctx, state.LastID, int64(req.PageSize))
+	if err != nil {
+		return models.FetchNextPaymentsResponse{}, fmt.Errorf("failed to get transactions: %w", err)
+	}
+
+	// Get pending transactions
+	pendingTransactions, _, _, err := p.client.GetPendingTransactions(ctx, "", int64(req.PageSize))
```

Comment:
`NextCursor` and `HasMore` from this function are not being used. How will we know if there is still data of this type to be polled?